From 0e7e16b8725153eb046e1c27774f81d866e264cb Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Sun, 15 Sep 2019 14:14:57 +0300
Subject: [PATCH 1/5] PRU: Fix comment to avoid fall through warning

gcc/ChangeLog:

2019-10-13  Dimitar Dimitrov  <dimitar@dinux.eu>

	* config/pru/pru.c (pru_print_operand): Fix comment.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 gcc/config/pru/pru.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/pru/pru.c b/gcc/config/pru/pru.c
index 16d1451262e..59b004b774f 100644
--- a/gcc/config/pru/pru.c
+++ b/gcc/config/pru/pru.c
@@ -1650,7 +1650,7 @@ pru_print_operand (FILE *file, rtx op, int letter)
 	  return;
 	case 'Q':
 	  cond = swap_condition (cond);
-	  /* Fall through to reverse.  */
+	  /* Fall through.  */
 	case 'R':
 	  fprintf (file, "%s", pru_comparison_str (reverse_condition (cond)));
 	  return;
-- 
2.20.1

